docs/grammar.txt

```diff
@@ -79,7 +79,7 @@ Break = 'break'
 Return = 'return' Expression?
 Throw = 'throw'
 EmitStatement = 'emit' FunctionCall
-VariableDefinition = (VariableDeclaration | '(' VariableDeclaration? (',' VariableDeclaration? )* ')' ) ( '=' Expression )?
+VariableDefinition = (VariableDeclaration | '(' VariableDeclaration (',' VariableDeclaration )* ')' ) ( '=' Expression )?
```

Comment:
Sorry, but this is not true. The following is valid code:
`(uint x, , uint y) = f()`.